### ISSUE TITLE
Consider reducing Klyde's damage output

### ISSUE THREAD

**Issue by zxcvbgfrtyui** [HelixNebulaStudio/RiseOfTheDead#321]
Klyde (the gunner in The Billies boss fight) currently deals 10 damage with each shot. Even though the firerate has been reduced significantly, it is still high enough that Klyde will kill players in seconds, with or without armour. This would be somewhat alright if the level requirement was high enough that any player meeting the requirement would have good ways of dealing with this, but at level 160, the only working solutions are a tactical bow with ricochet so that it can rebound around corners into Klyde or spamming Sticky Grenades in Klyde's direction and hoping that they kill him fast enough. Other available strategies are simply too slow to be done before either the boss room ends or Klyde kills the player.

**Comment by Leroy8888** [comment]
I am pretty sure the player is supposed to trigger Klyde's attack and then hide. He will keep firing is he start until his clip is empty, in which he will reload and grant the player a bit of time to deal some damage. Likely the reason why there are many metal barricades around in the arena

**Comment by zxcvbgfrtyui** [comment]
From what I've seen in recent testing, Klyde doesn't keep firing once the player goes back into cover

**Comment by zxcvbgfrtyui** [comment]
Also, Klyde will move fast enough that he will flank the player if the player is on the lower floor and hiding behind cover.

**Comment by Leroy8888** [comment]
Hmm odd, I've never seen Klyde stop firing until he empties his clip. I did attempt this without armor and removed all my mods and did succeed. It was difficult but possible. I could've brought healing like medkit to heal while taking cover.

Klyde did flank me at times, so I had to pay attention and jump between cover or carefully use the cover the keep myself protecte, but it was still doable despite me taking some damage.
